homepage/homepage/app/docs/[framework]/[...slug]/project-setup/react.mdx

```diff
@@ -43,28 +15,32 @@ Currently, the recommended pattern to set up a React app with Jazz is to create
     <CodeGroup>
     {/* prettier-ignore */}
     ```tsx
-    import { createJazzReactApp } from "jazz-react";// old
-
-    const Jazz = createJazzReactApp();// old
-    export const { useAccount, useCoState } = Jazz;// old
-
-    import { PasskeyAuthBasicUI, usePasskeyAuth } from "jazz-react";
+    import { JazzProvider } from "jazz-react";// old
+    import { MyAppAccount } from "./schema";
 
     function JazzAndAuth({ children }: { children: React.ReactNode }) {
         const [passkeyAuth, passKeyState] = usePasskeyAuth({ appName });
 
         return (
             <>
-                <Jazz.Provider
+                <JazzProvider
                     auth={passkeyAuth}
                     peer="wss://cloud.jazz.tools/?key=you@example.com"
+                    AccountSchema={MyAppAccount}
                 >
                     {children}
-                </Jazz.Provider>
+                </JazzProvider>
                 <PasskeyAuthBasicUI state={passKeyState} />
             </>
         );
     }
+
+    // Register the Account schema for type safety
```

Comment:
Let's be more specific here

```suggestion
    // Register the Account schema so `useAccount` returns our custom `MyAppAccount`
```

Reply:
And here: https://github.com/garden-co/jazz/pull/1104/files#diff-7546d93c0714a65185a6409209e5412cdd7ab1b64006bd2cf86fb5bdd4448e47R47

We need to change the sentence: remove "extracted and exported" and just call them Jazz hooks or sth like that